react/week2/my-todos/src/index.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+
+const todos = [
+  {
+    "id": 1,
+    "description": "Get out of bed",
+    "checked": false
+  },
+  {
+    "id": 2,
+    "description": "Brush teeth",
+    "checked": false
+  },
+  {
+    "id": 3,
+    "description": "Eat breakfast",
+    "checked": false
+  },
+  {
+   "id": 4,
+   "description": "Learn React",
+   "checked": false
+  },
+  {
+    "id": 5,
+    "description": "Do homework",
+    "checked": false
+  }
+ ];
+ 
+class TodoList extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = { 
+      todoList: []
+    } 
+    this.addTodo = this.addTodo.bind(this)
+    this.deleteTodo = this.deleteTodo.bind(this)
+    this.handleCheckbox = this.handleCheckbox.bind(this)
+  }
+
+  addTodo() {  
+    let randomNumber = Math.floor(Math.random() * todos.length);
+    this.setState({ todoList: this.state.todoList.concat(todos[randomNumber]) })   
+  }
```

Comment:
You can avoid the need to bind this to your methods if you use arrow functions instead! 
```suggestion
  addTodo = () => {  
    let randomNumber = Math.floor(Math.random() * todos.length);
    this.setState({ todoList: this.state.todoList.concat(todos[randomNumber]) })   
  }
```